products/fedora/product.yml

```diff
@@ -12,8 +12,6 @@ pkg_manager: "dnf"
 init_system: "systemd"
 
 # EFI and non-EFI configs are stored in same path, see https://fedoraproject.org/wiki/Changes/UnifyGrubConfig
-grub2_boot_path: "/boot/grub2"
-grub2_uefi_boot_path: "/boot/grub2"
```

Comment:
the comment above probably needs to be moved as well

Reply:
Removed the comment, as what used to be news became the new norm.